modules/postman-templating/src/template-client.ts

```diff
@@ -189,6 +187,22 @@ export class TemplateClient {
      * removeEmptyLines
      */
     if (options.removeEmptyLines) {
+      // Remove empty tags so that collapsing of line breaks will work even if line contains empty tags
+      // Trim all empty spaces between tags so that they can be recognised as empty. E.g. <b> </b>
+      result = result.replace(/>\s+</g, '><')
```

Comment:
what about other whitespace characters

Reply:
Was thinking of collapsing all whitespace (tabs, etc) with the `\s` metacharacter, in case it was inserted by some other text editor the template author was using. But I realised something like this wouldn't get collapsed:

```
<b>
</b>
hello
```

Because it becomes the following:
```
<b><br></b>
<br>
Hello
```

Not sure how extensive should the stripping of empty tags be done though...